osu.Game/Screens/Menu/LogoVisualisation.cs

```diff
@@ -34,7 +35,12 @@ public class LogoVisualisation : Drawable, IHasAccentColour
         /// <summary>
         /// The maximum length of each bar in the visualiser. Will be reduced when kiai is not activated.
         /// </summary>
-        private const float bar_length = 600;
+        private const float bar_length_const = 600;
+
+        /// <summary>
+        /// The calculated max length of each bar in the osu!logo visualizer (calculated from the audio value)
+        /// </summary>
+        private static float barLength => Convert.ToSingle(bar_length_const * audio.Volume.Value);
```

Comment:
is not considering music volume, only master

Reply:
@peppy I had done this as the issue stated master, but if you want music only I can change that

Reply:
> is not considering music volume, only master 
(in addition) @peppy  I feel that the master would be a better measurement as it is the default volume that is changed and it does include the music volume. I'm happy to change this if you disagree